.gitignore

```diff
@@ -16,3 +16,4 @@ debian/*.log
 debian/*.substvars
 debian/files
 debian/debhelper-build-stamp
+.idea/
```

Comment:
If you use ` /.idea/ ` it'll only look for that folder in the root of the project, which makes Git a bit faster. See for example https://github.com/linz/linz-lds-bde-schema/pull/262/files.